taigaProject/frontend/src/App.js

```diff
@@ -18,6 +19,9 @@ function App() {
   return (
     <div className="container-full">
       <Router>
+        <ProSidebarProvider>
+          <SidebarMenu />
```

Comment:
Has the relative behavior between the SidebarMenu and the contents of each page been tested? I see it has been moved from within each component to the outside.

Reply:
It has. For our relatively simple use-case scenarios, it most definitely will never create any issues.